### PR TITLE
Feat 3 video listing

### PR DESCRIPTION
**Pull request by SinchanaMS**
The following features have been added to this PR:

1. Theme Provider
2. Header and Sidebar
3. Video Database
4. Video Listing Page

### CODE REVIEW

File: src/pages/Explore/Explore.jsx [hunk 1]
```diff
@@ -1,3 +1,48 @@
+import axios from "axios";
+import { useState } from "react";
+import Sidebar from "../../components/Sidebar";
+import { useTheme } from "../../contexts/ThemeContext";
+import "../../styles/videolisting.css";
+
```
Comment (Debarshi95) [inline src/pages/Explore/Explore.jsx:6]: Please use absolute imports

File: src/pages/Explore/Explore.jsx [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+    try {
+      if (response.status === 200) {
+        setVideoList(response.data.videos);
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  videos();
```
Comment (Debarshi95) [inline src/pages/Explore/Explore.jsx]: This should be called inside useEffect no?

**Review by Debarshi95 — APPROVED** [review]
Good work . An important change required with videos function. Do wrap it in useEffect.

**Review by yashodaAgarwal — APPROVED** [review]
Overall code looks good. Can change some larger px values to rem if not needed for some particular purpose. Code is very structured and neat.Liked on how well you have used variables for color all over the code.

**Review by m26unkwn — APPROVED** [review]
Love the dark mode and code is clean.

good use of spacing

**Comment by SinchanaMS** [comment]
Thank you for the reviews, merging to dev.

**Pull request merged**